pkg/cli/cmd/radinit/aws.go

```diff
@@ -123,3 +169,15 @@ func (r *Runner) buildAWSRegionsList(listRegionsOutput *ec2.DescribeRegionsOutpu
 
 	return regions
 }
+
+func (r *Runner) selectAWSCredentialKind() (string, error) {
+	credentialKinds := r.buildAWSCredentialKind()
+	return r.Prompter.GetListInput(credentialKinds, selectAWSCredentialKindPrompt)
+}
+
+func (r *Runner) buildAWSCredentialKind() []string {
```

Comment:
I guess we can also change the name of the function. Are we building something here? Is this the convention we are using? Isn't it more like `awsCredentialKinds`?

Reply:
this is format we used in azure, we are using the similar format in azure provider.